pkg/cluster/install.go

```diff
@@ -102,7 +102,7 @@ func (m *manager) adminUpdate() []steps.Step {
 	if isEverything {
 		toRun = append(toRun,
 			steps.Action(m.ensureGatewayUpgrade),
-			steps.Action(m.rotateACRTokenPassword),
+			steps.Condition(m.rotateAndValidateACRTokenPassword, 5*time.Minute, true),
```

Comment:
Don't we need to ensure the operator is running before moving forward with the rotation? You can make sure the operator is ready using the `aroDeploymentReady` Condition function.